src/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  font-size: 13px;
+  font-family: Arial, sans-serif;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+h2 {
+  font-size: 15px;
+  margin-top: 1em;
+  margin-bottom: 1em;
+  font-weight: normal;
+}
+
+a {
+  color: green;
+}
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice

Reply:
I tried added stylizing tags, but the tasks is doesn't work on Backstop e.t.c, and Misha said sent this task with this solution.

Reply:
I tried added stylizing tags, but the task is doesn't work on Backstop e.t.c, and Misha said sent us this task with this solution.

Reply:
don't mind about backstop, add styles using classes